### DISCUSSION TITLE
Page not found if it has `__` as prefix?

### DISCUSSION THREAD

**Discussion by mr-cheffy** [community/community#81926] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

[This page](https://snowball-lang.github.io/snowball/std/__winter_wonderland__.html) says it's not found but [this one](https://snowball-lang.github.io/snowball/std/io.html) is found. I've also checked that https://github.com/snowball-lang/snowball/blob/stdlib-docs/std/__winter_wonderland__.html does indeed exist. So, I'm just wondering, does GitHub ignore `double underscore`-ed prefixed pages?

**Comment by dostogircse171** [comment]
Repository settings or .gitignore files might have rules that affect how certain files are treated. Make sure there aren't any settings that could be causing this issue.

**Reply by mr-cheffy** [reply]
already checked and no .gitignore, so... this must be a GitHub security thing.

**Reply by dostogircse171** [reply]
Seems like that. Never realized this issue exists in Git unless I found your discussion. This is an interesting problem. For now I think it better to rename the file and in the meantime try to contact Git support.

**Comment by mouadelfaiz** [comment]
intersted

**Comment by yoannchaudet** [comment]
This is a [Jekyll](https://jekyllrb.com/) behavior to ignore files starting with an underscore. Jekyll is the default static site generator we use to build your site if you did not specify or ask for anything else.

Since you are using HTML files directly in your repo, you probably want to add a `.nojekyll` (empty) file at the root of your repository. As an alternative you can change the source of your site (in the settings) to GitHub Actions (instead of branch) and select our Static workflow template.
